packages/backend/src/plugins/catalog.ts

```diff
@@ -35,8 +35,14 @@ export default async function createPlugin(
   for (const config of env.config.getOptionalConfigArray(
     'catalog.providers.okta',
   ) || []) {
-    const groupProvider = OktaGroupEntityProvider.fromConfig(config, env);
-    const userProvider = OktaUserEntityProvider.fromConfig(config, env);
+    const groupProvider = OktaGroupEntityProvider.fromConfig(config, {
+      ...env,
+      namingStrategy: 'kebab-case-name',
```

Comment:
isn't the name ´strip-domain-email´ as defined in the last file?

Reply:
Groups don't have an email in okta so I'm using the profile.name field for group.